README.md

```diff
@@ -18,59 +29,59 @@ If you're new here, here are some things you should know:
 
-### Origins
+Once you've succesfully installed ruby. You may proceed to the next section!
 
+### Install Node
+You must install the version of node specified in `.nvmrc`. It is highly recommended to use [nvm](https://github.com/nvm-sh/nvm).
 
-This project took what we built for the [Portland Diaper Bank in 2016](https://github.com/rubyforgood/pdx_diaper) and turned it into a multitenant application, something that all essentials banks can use. We re-used models, code and other documentation where applicable as well as implemented new features and functionality requested by the prime stakeholder (PDXDB). We're super excited to have had Rachel Alston, the director of the Portland Diaper Bank, attending our event in 2017, providing guidance and giving us the best chance of success!
+You can follow the guide [here](https://heynode.com/tutorial/install-nodejs-locally-nvm/) for setting up node.
 
+*NOTE: It's possible that Node version 12 may cause you problems, see issue #751. Node 10 or 11 seem to be fine.*
 
-## Development
+### Install Postgres
+You must install postgres and run the database locally. Instructions differ depending on the operating system.
 
-### Installation Instructions
+Follow one of these guides to install postgres:
 
-The `installation.md` file ([https://github.com/rubyforgood/human-essentials/blob/main/installation.md](https://github.com/rubyforgood/human-essentials/blob/main/installation.md)) has detailed instructions for installation and configuration of an Ubuntu host to run this software. Although there is not a document for Mac OS, it may be helpful for that as well.
+###### MacOSX
+https://www.digitalocean.com/community/tutorials/how-to-use-postgresql-with-your-ruby-on-rails-application-on-macos
 
-### Ruby Version
-This app uses Ruby version 3.0.2, indicated in `/.ruby-version` and `Gemfile`, which will be auto-selected if you use a Ruby versioning manager like `rvm`, `rbenv`, or `asdf`.
+###### Ubuntu
+TODO
 
-### Yarn Installation
-If you don't have Yarn installed, you can install with Homebrew on macOS `brew install yarn` or visit [https://yarnpkg.com/en/docs/install](https://yarnpkg.com/en/docs/install). Be sure to run `yarn install` after installing Yarn. NOTE: It's possible that Node version 12 may cause you problems, see issue #751. Node 10 or 11 seem to be fine.
+###### Windows
+TODO
 
-### Install dependencies using Yarn
-Run `yarn` to install project dependencies.
+After completing this step, you should have both ruby, node, and postgres installed. Now you can proceed to the next step which is to setup the application.
 
-### Create your .env with database credentials
-Be sure to create a `.env` file in the root of the app that includes the following lines (change to whatever is appropriate for your system):
+### Run Setup
+Run the following command should run all the neccessary steps to get up and running.
 ```
-PG_USERNAME=username
-PG_PASSWORD=password
+bin/setup
```

Comment:
Following the guide there is no mention of installing yarn. When I ran the `bin/setup`  script I got an error related to db/seed.

After I installed yarn (Classic yarn because I couldn't get yarn working with corepack for v12) the bin/setup script ran without trouble

Reply:
Nice catch! Would you mind adding a step to install yarn prior to this step?

Side note - Argh! I can't wait to remove node dependency!

Reply:
Sure I'll do it! (I won't be able to add it before friday though)

Reply:
Solved by https://github.com/rubyforgood/human-essentials/pull/2970/commits/81eddeb85b3afd6857b63a3085500b5809d5c945